content/tracing/setup/javascript.md

```diff
@@ -0,0 +1,163 @@
+
+### Configuration Options
+
+Options can be configured as a parameter to the `init()` method or as environment variables.
+
+| Config        | Environment Variable         | Default   | Description |
+| ------------- | ---------------------------- | --------- | ----------- |
+| debug         | DD_TRACE_DEBUG               | false     | Enable debug logging in the tracer. |
+| service       | DD_SERVICE_NAME              |           | The service name to be used for this program. |
+| hostname      | DD_TRACE_AGENT_HOSTNAME      | localhost | The address of the trace agent that the tracer will submit to. |
+| port          | DD_TRACE_AGENT_PORT          | 8126      | The port of the trace agent that the tracer will submit to. |
+| env           | DD_ENV                       |           | Set an application’s environment e.g. `prod`, `pre-prod`, `stage`. |
+| tags          |                              | {}        | Set global tags that should be applied to all spans. |
+| flushInterval |                              | 2000      | Interval in milliseconds at which the tracer will submit traces to the agent. |
+| experimental  |                              | {}        | Experimental features can be enabled all at once using boolean `true` or individually using key/value pairs. Available experimental features: `asyncHooks`. |
+| plugins       |                              | true      | Whether or not to enable automatic instrumentation of external libraries using the built-in plugins. |
+
+## OpenTracing
+
+This library is OpenTracing compliant. Use the [OpenTracing API](https://doc.esdoc.org/github.com/opentracing/opentracing-javascript/) and the Datadog Tracer (dd-trace) library to measure execution times for specific pieces of code. In the following example, a Datadog Tracer is initialized and used as a global tracer:
+
+```js
+const tracer = require('dd-trace').init()
+const opentracing = require('opentracing')
+
+opentracing.initGlobalTracer(tracer)
+```
+
+**NOTE: The tracer returned by `opentracing.globalTracer()` only
+contains OpenTracing specific methods.**
+
+**NOTE: When using OpenTracing, context propagation is not handled
+automatically.**
+
+## Integrations
+
+APM provides out-of-the-box instrumentation for many popular frameworks and libraries by using a plugin system. By default all built-in plugins are enabled. This behavior can be changed by setting the `plugins` option to `false` in the [configuration options](#configuration-options).
+
+Built-in plugins can be enabled by name and configured individually:
+
+```js
+const tracer = require('dd-trace').init({ plugins: false })
+
+// enable express integration
+tracer.use('express')
+
+// enable and configure postgresql integration
+tracer.use('pg', {
+  service: 'pg-cluster'
+})
+```
+
+See [compatibility](#compatibility) for the list of supported integrations.
+
+### Compatibility
```

Comment:
Do we want to add a sentence telling people to open a github ticket if they want to request a new integration? @rochdev @palazzem

Reply:
I think being an open source project this is implied, but I could add a note.

Reply:
Let's add a sentence about that

Reply:
I added a sentence in the existing introduction. Let me know if that works!